pants

```diff
@@ -13,19 +13,18 @@
 
 set -eou pipefail
 
+# NOTE: To use an unreleased version of Pants from the pantsbuild/pants master branch,
+#  locate the master branch SHA, set PANTS_SHA=<SHA> in the environment, and run this script as usual.
+#
+# E.g., PANTS_SHA=725fdaf504237190f6787dda3d72c39010a4c574 ./pants --version
+
 PYTHON_BIN_NAME="${PYTHON:-unspecified}"
 
-# Set one of these to specify a non-standard location for this script to read the pants version from.
+# Set this to specify a non-standard location for this script to read the Pants version from.
 # NB: This will *not* cause Pants itself to use this location as a config file.
 #     You can use PANTS_CONFIG_FILES or --pants-config-files to do so.
-PANTS_INI=${PANTS_INI:-pants.ini}
 PANTS_TOML=${PANTS_TOML:-pants.toml}
 
-# NOTE: To use an unreleased version of Pants from the pantsbuild/pants master branch,
-#  locate the master branch SHA, set PANTS_SHA=<SHA> in the environment, and run this script as usual.
-#
-# E.g., PANTS_SHA=725fdaf504237190f6787dda3d72c39010a4c574 ./pants --version
-
```

Comment:
Did you intend to delete the PANTS_SHA documentation?

Reply:
Nope, I simply moved it up to the top of the file to make it more likely people see it.

Reply:
Ah I missed that.